.ci/xtask/src/packages/carl.rs

```diff
@@ -127,15 +130,57 @@ pub mod distribution {
             fs::create_dir_all(cleo_out_dir)?;
             
             for arch in architectures {
-                crate::packages::cleo::build::build_release(arch.to_owned())?;
-                let cleo_build_dir = crate::packages::cleo::build::out_dir(arch.to_owned());
+                crate::packages::cleo::distribution::cleo_distribution(arch.to_owned())?;
+                let cleo_build_dir = crate::tasks::distribution::out_arch_dir(arch.to_owned());
 
-                let cleo_arch_dir = out_dir.join(Package::Cleo.ident()).join(format!("{}-{}", Package::Cleo.ident(), arch.triple()));
+                let cleo_arch_dir = out_dir.join(Package::Cleo.ident());
                 fs::create_dir_all(&cleo_arch_dir)?;
-                
+
+                let tar_file_name = bundle::out_file(Package::Cleo, *arch);
+
+                let cleo_tar_file_name = tar_file_name.file_name().context(format!("Could not extract file name {}", &tar_file_name.display()))?;
+
+                fs_extra::file::copy(
+                    cleo_build_dir.join(&tar_file_name),
+                    &cleo_arch_dir.join(cleo_tar_file_name),
+                    &fs_extra::file::CopyOptions::default()
+                        .overwrite(true)
+                )?;
+            }
+
+            Ok(())
+        }
+    }
+
+    mod edgar {
+        use anyhow::Context;
+        use clap::ValueEnum;
+        use crate::tasks::distribution::bundle;
+        use super::*;
+
+        #[tracing::instrument]
+        pub fn get_edgar(out_dir: &PathBuf) -> crate::Result {
+
+            let architectures = Arch::value_variants().iter()
+                .filter(|&&arch| arch != Arch::Wasm).collect::<Vec<_>>();
```

Comment:
We decided against a constant, because we did not want to go with String/str. Instead we implemented two iterators which can now be changed as needed, if the architectures change in the future. Also we decided to implement two iterators directly, since in the future EDGAR and CLEO might differ in their architectures.